src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -108,16 +109,45 @@ define(function (require, exports, module) {
          *
          * @private
          */
-        _addAssetClickHandler: function (layer) {
+        _addAssetClickHandler: function () {
             var document = this.props.document,
+                selectedLayers = document && document.layers.selected,
                 documentExports = this.state.documentExports,
-                layerExports = documentExports && documentExports.layerExportsMap.get(layer.id),
-                existingScales = (layerExports && collection.pluck(layerExports, "scale")) || Immutable.List(),
+                exportsList,
+                layer;
+
+            if (selectedLayers && selectedLayers.size > 0) {
+                // temp until we find a better way to handle more layers...
+                layer = selectedLayers.first();
+                exportsList = layer && documentExports && documentExports.layerExportsMap.get(layer.id);
+            } else {
+                exportsList = documentExports && documentExports.rootExports;
+            }
+                
+            var existingScales = (exportsList && collection.pluck(exportsList, "scale")) || Immutable.List(),
                 remainingScales = collection.difference(ExportAsset.SCALES, existingScales),
                 nextScale = remainingScales.size > 0 ? remainingScales.first() : null,
-                nextAssetIndex = (layerExports && layerExports.size) || 0;
+                nextAssetIndex = (exportsList && exportsList.size) || 0;
+
+            var layers = layer ? Immutable.List([layer]) : null;
```

Comment:
(I know this is temporary, but I can't help myself.) The following is better: `Immutable.list.of(layer)`.